hosting-home/modules/ROOT/pages/index.adoc

```diff
@@ -73,6 +74,7 @@ Not all Anypoint Platform components are supported in all cloud environments. Th
 | Anypoint DataGraph | Y | Y | N | N
 | Anypoint Exchange | Y | Y | Y | Y
 | Flow Designer | Y | Y | N | N
+| Anypoint MQ | Y | N | Y <<mq-support,^***^>> | N
```

Comment:
This should be a "Y" for MQ in EU Control Plans

Reply:
ok